Classes/Login/LoginSplashViewController.swift

```diff
@@ -24,6 +25,17 @@ final class LoginSplashViewController: UIViewController, GithubSessionListener {
     @IBOutlet weak var activityIndicator: UIActivityIndicatorView!
     private weak var safariController: SFSafariViewController? = nil
 
+    @available(iOS 11.0, *)
+    private var authSession: SFAuthenticationSession? {
+        get {
+            return _authSession as? SFAuthenticationSession
```

Comment:
Sorry I'm probably just being inept, why are we having to use this getter/setter pattern rather than just using the var `authSession` in the first place?

Reply:
@available can not be used on stored properties. This solution allows the type to be defined.

I figured it is better than having to type cast.

```swift
(self?.authsession as! SFAuthenticationSession).start()
```

Another solution would be using a lazy var, but I thought the get/set pattern was most readable.

```swift
@available(iOS 11.0, *)
lazy var authSession: SFAuthenticationSession? = { return nil }()
```